typings/index.d.ts

```diff
@@ -3138,23 +3213,48 @@ declare module 'discord.js' {
 
   type MessageAdditions = MessageEmbed | MessageAttachment | (MessageEmbed | MessageAttachment)[];
 
+  interface MessageActionRowOptions extends BaseMessageComponentOptions {
+    type: 'ACTION_ROW';
+    components?: MessageComponentResolvable[];
+  }
+
   interface MessageActivity {
     partyID: string;
     type: number;
   }
 
+  interface MessageButtonOptions extends BaseMessageComponentOptions {
+    customID?: string;
+    disabled?: boolean;
+    emoji?: RawEmoji;
+    label?: string;
+    style?: MessageButtonStyle;
```

Comment:
This seems to already support `MessageButtonStyles` (the enum), so might as well declare it:
```suggestion
    style?: MessageButtonStyle | MessageButtonStyles;
```